src/logging/WriterBackend.h

```diff
@@ -60,6 +60,7 @@ class WriterBackend : public threading::MsgThread
 		 */
 		const char* path;
 
+		const char* logdir;
```

Comment:
I do not think that this should be added like this. A lot of writers will not have a concept of a directory. Might it work to fold this into the path in scriptland beforehand?

Reply:
Actually - you are not even using this later, are you? The way in which you retrieve logdir later is via the config map - which is an approach that I can get behind. That is reasonable.

Reply:
I believe you are right, and have removed that line.